### PR TITLE
Fixes #15711: Rudder api client expect python3 which is not available by default on rhel7

### PR DESCRIPTION
**Pull request by Fdall**
https://issues.rudder.io/issues/15711

### CODE REVIEW

File: rudder-api-client/SPECS/rudder-api-client.spec [hunk 1]
```diff
@@ -61,6 +61,11 @@ Command line tools and python libraries to call Rudder.
 %prep
 %setup -c
 
+# rhel7 does not have python3 so we force python2 instead
+%if 0%{?rhel} == 7
+find . -type f | xargs sed -i '1,1s|#!/usr/bin/python3|#!/usr/bin/python2|'
+%endif
```
Comment (peckpeck) [inline rudder-api-client/SPECS/rudder-api-client.spec]: the requires and dependencies must also be updated

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
OK, squash merging this PR

**Pull request merged**